src/tests/ftest/server/metadata_svc_ops.py

```diff
@@ -0,0 +1,141 @@
+"""
+  (C) Copyright 2022-2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import statistics
+import time
+
+from apricot import TestWithServers
+from avocado.core.exceptions import TestFail
+from general_utils import DaosTestError
+from thread_manager import ThreadManager
+
+
+class DuplicateRpcDetection(TestWithServers):
+    """Compare metadata handling performance between pools with and without duplicate rpc
+       detection feature enabled.
+
+    Test Class Description:
+        Create pools run test with and without duplicate rpc detection feature and verify time
+        consuming with metadata workload before and after svc ops full.
+
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        """Set Up DuplicateRpcDetectionTest"""
+        super().setUp()
+        self.dmg = self.get_dmg_command()
+
+    def metadata_workload_test(self, pool=None, num_of_cont=1, workload_cycles=5000, test_loops=1):
+        """To create single container and perform metadata workload tests.
+
+        Args:
+            pool (str): pool handle to create container. Defaults to None.
+            num_of_cont (int): Number of container to be created. Defaults to 1.
+            workload_cycles (int): Number of metadata workload test cycles per test loop.
+                Defaults to 5000.
+            test_loops (int): Number of metadata workload test loops. Defaults to 1.
+
+        Returns:
+            test_time: List of time consumed per test loops of metadata workload test cycles.
+
+        """
+        test_time = []
+        try:
+            container = self.get_container(pool, create=False)
+            if container.daos:
+                container.daos.verbose = False
+            container.create()
```

Comment:
It's better to do this upfront

```suggestion
            daos = self.get_daos_command()
            daos.verbose = False
            container = self.get_container(pool, daos=daos)
```